AUTHORS

```diff
@@ -1,6 +1,11 @@
+Alan Bishop <abishop@redhat.com>
+Arx Cruz <arxcruz@redhat.com>
 Brendan Shephard <59072170+bshephar@users.noreply.github.com>
 Chandan Kumar <raukadah@gmail.com>
 Rabi Mishra <ramishra@redhat.com>
 Ronelle Landy <rlandy@redhat.com>
 Steve Baker <sbaker@redhat.com>
+Takashi Kajinami <kajinamit@users.noreply.github.com>
```

Comment:
are these duplicates intentional? for Takashi possibly since the email is different

Reply:
probably some rebase went wrong...